sorc/gfs_bufr.fd/calpreciptype.f

```diff
@@ -77,7 +77,7 @@ SUBROUTINE CALPRECIPTYPE(kdt,nrcm,im,ix,lm,lp1,randomno,      &
       ALLOCATE ( RH(LM),TD8(LM),TWET8(LM) )
 
 ! Create look up table
-      call gfuncphys
+!     call gfuncphys
```

Comment:
Remove inactive code?

Reply:
I can remove it once it's in `dev_v16` (for GFSv16.3+). I already handed off the v16.2.1 tag to NCO so I shouldn't touch `feature/ops-wcoss2` at this point without changes in ops.